src/app/dynamic-forms/cinchy-dynamic-forms.component.ts

```diff
@@ -377,12 +377,16 @@ export class CinchyDynamicFormsComponent implements OnInit, OnChanges, OnDestroy
         async (formFieldsMetadata) => {
 
           let selectedLookupRecord = this.lookupRecordsList.find(_ => _.id == this.rowId);
-          await this._formHelperService.fillWithFields(this.form, this.rowId as string, this.formMetadata, formFieldsMetadata, selectedLookupRecord,tableEntitlements);
-          await this._formHelperService.fillWithData(this.form, this.rowId as string, selectedLookupRecord, null, null, null, this.afterChildFormEdit.bind(this));
+         
+          setTimeout(async () => {
+            await this._formHelperService.fillWithFields(this.form, this.rowId as string, this.formMetadata, formFieldsMetadata, selectedLookupRecord,tableEntitlements);
+            await this._formHelperService.fillWithData(this.form, this.rowId as string, selectedLookupRecord, null, null, null, this.afterChildFormEdit.bind(this));
+            this.enableSaveBtn = true;
+          }, 1000);
```

Comment:
does this need to be 1000ms? if we're making this async does it make sense to just fire it off at 0ms?

Reply:
Hi @alvin-h  I have change 1000ms to 0ms and resolved above comment.